docs/admin/kubeadm-config.md

```diff
@@ -0,0 +1,36 @@
+---
+approvers:
+- mikedanese
+- luxas
+- jbeda
+title: Kubeadm config
+notitle: true
+---
+{% capture overview %}
+# Kubeadm config
+Beginning with v1.8.0, kubeadm uploads the configuration of your cluster to a ConfigMap called 
+`kubeadm-config` in the `kube-system` namespace, and later reads that configuration when upgrading 
+in order to configure system components correctly. This provides a seamless user experience.
+
```

Comment:
Suggested change:
... in the `kube-system` namespace, and later reads the ConfigMap when upgrading. This enables correct configuration of system components, and provides a seamless user experience.